pulsar-client/pom.xml

```diff
@@ -214,7 +214,8 @@
 
     <dependency>
       <groupId>it.unimi.dsi</groupId>
-      <artifactId>fastutil</artifactId>
+      <artifactId>fastutil-core</artifactId>
+      <version>${it-unimi-dsi-fastutil.version}</version>
```

Comment:
The version could be omitted here by adding an entry to the root `pom.xml` `dependencyManagement`. You could add `fastutil-core` to follow after the existing `fastutil` dependency in dependencyManagement.

Reply:
good idea, updated.